### ISSUE TITLE
CSharp grammar contains lexical action which uses undefined variable

### ISSUE THREAD

**Issue by mrtxnk** [antlr/grammars-v4#328]
The action of COLON token in [CSharpLexer.g4](https://github.com/antlr/grammars-v4/blob/master/csharp/CSharpLexer.g4#L178) references "_input" variable, which doesn't exist, so the generated CSharpLexer.cs won't compile. 
The variable is referenced at [line 184](https://github.com/antlr/grammars-v4/blob/master/csharp/CSharpLexer.g4#L184) and [line 186](https://github.com/antlr/grammars-v4/blob/master/csharp/CSharpLexer.g4#L186).

Additional info: I use Visual Studio 2015 Update 1 and the official ANTLR C# Target 4.5.2 (so not the alternative one by Sam Harwell).

**Comment by KvanTTT** [comment]
I was using last alpha version of ANTLR C# runtime from NuGet: https://www.nuget.org/packages/Antlr4/4.5.0-alpha003

**Comment by ericvergnaud** [comment]
The value can be accessed using the property "InputStream"

**Comment by xied75** [comment]
@ericvergnaud you are fast. So we should update the g4 file here for the COLON action then?

**Comment by ericvergnaud** [comment]
Yes. Note that you might also need to change La to LA, I think I aligned 4.7 to Java

**Comment by xied75** [comment]
Right, I'll make a PR if no others beat me first. BTW, if we now have this Getter and I see Java has *getInputStream*, then Java runtime should also set _input **private**, is that right?

**Comment by KvanTTT** [comment]
@xied75 if you want to use a Standart runtime, create a copy of lexer file because of Standart differs from SHarwell.

**Comment by ericvergnaud** [comment]
Not sure about what needs to be done for Java.
That's with Terence.

Envoyé de mon iPhone

**Comment by ericvergnaud** [comment]
Not sure what you mean here (Sam's Lexer is already different)

Envoyé de mon iPhone

**Comment by jpwiddy** [comment]
This is also an issue when using the ECMAScript.g4 grammar. There are multiple references to `_input.Get` and `_input.Lt` and `_input.La`, whereas "InputStream" only has `LA` defined.

**Comment by Marti2203** [comment]
Hi! Is this issue fixed?
